index.js

```diff
@@ -12,10 +12,12 @@ class Build {
   error (message, url) {return update(this.meta, message, url, 'error')}
 }
 
+const default_git_url = "https://api.github.com"
+
 const update = (build, message, url, status) => new Promise((resolve, reject) => {
   axios({
     method: 'POST',
-    url: `https://api.github.com/repos/${build.repo}/statuses/${build.sha}`,
+    url: `${build.ghe ? `${build.ghe}/api/v3` : default_git_url }/repos/${build.repo}/statuses/${build.sha}`,
```

Comment:
Where does `ghe` come from? 🤔

Reply:
@rahulgandhi can you respond to this?

Reply:
Well, I think `ghe` comes from `Build` instance `meta`.

E.g.:
```js
const data = {
  ...
  ghe: '...',
}

/* Create a build */
const build = new Build(data)
```

Although, IMAO `ghe` is not obvious. Maybe `ghEnterpriseUrl`?

Reply:
Agree, one should be allowed to configure the entire thing. In other projects i've seen them separated out as:

GH_PREFIX and GH_URL (as env vars, for example). Makes for easier separate consumption.

GH_PREFIX being `/api/v3` and GH_URL being `https://github.mydomain.com`

Reply:
Hey @rahulgandhi, do you have time to update this?